### PR TITLE
Fix: chunk MGET and DELETE calls on multiple keys

### PR DESCRIPTION
**Pull request by psrok1**
When there are lots of tasks in Karton, `get_all_tasks` and `delete_tasks` fail due to amount of keys passed to `MGET` and `DELETE` calls.

### CODE REVIEW

File: karton/core/backend.py [hunk 1]
```diff
@@ -257,16 +258,18 @@ def get_tasks(self, task_uid_list: List[str]) -> List[Task]:
             if task_data is not None
         ]
 
-    def get_all_tasks(self) -> List[Task]:
+    def get_all_tasks(self, chunk_size=1000) -> List[Task]:
```
Comment (nazywam) [inline karton/core/backend.py:261]:
```suggestion
    def get_all_tasks(self, chunk_size: int = 1000) -> List[Task]:
```

**Review by nazywam — APPROVED** [review]

**Comment by nazywam** [comment]
Did you perform any tests on the chunk size? Like 100 vs 10k vs 100k? 

Looks OK overall

**Comment by nazywam** [comment]
I'm also wondering whether we shouldn't apply the same pattern to `get_tasks`

**Comment by psrok1** [comment]
I haven't done tests on different chunk size, got that one from contributor's suggestion.

Chunking on `get_tasks` might be actually a nice addition in case there are lots of tasks in inspected task queue (used by get_task_queue)

**Pull request merged**